bin/cli.js

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+  [ '--karma-config <value>',    'Karma config file'],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+for (const level in karmaLogLevels) {
+  karmaLogLevels[level][0] = karma.constants[karmaLogLevels[level][0]];
```

Comment:
You reassign `karmaLogLevels` right away, can't we put those values directly in `karmaLogLevels` or `karma.constants` is somehow dynamic?